app/package.json

```diff
@@ -4,16 +4,14 @@
   "description": "Oran.js JavaScript by Orange Cats",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint",
+    "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node app/index.js",
-    "build": "gulp build",
-    "lint": "eslint . --ext .js"
+    "build": "gulp build"
```

Comment:
It looks like in your conflict resolution that you lost some changes from other PRs like the eslint additions.